beacon-chain/state/state-native/setters_payload_header.go

```diff
@@ -19,6 +20,22 @@ func (b *BeaconState) SetLatestExecutionPayloadHeader(val interfaces.ExecutionDa
 	}
 
 	switch header := val.Proto().(type) {
```

Comment:
We might want to rename `SetLatestExecutionPayloadHeader` to `SetLatestExecutionData`